system/jlib/jcomp.cpp

```diff
@@ -129,8 +129,8 @@ static void doSetCompilerPath(const char * path, const char * includes, const ch
         libs = LIB_DIR[targetCompiler];
     if (verbose)
     {
-        PrintLog("Include directory set to %s", includes);
-        PrintLog("Library directory set to %s", libs);
+        LOG(MCoperatorInfo, "Include directory set to %s", includes);
```

Comment:
@richardkchapman @jakesmith 
what do you think about logging like this.  It is true that the operator configures the path, but I don't think they want to be notified about it.  I think this is probably closer to DBGLOG.

Reply:
agree. DBGLOG seems about right.

Reply:
and true of the other LOG(MCoperatorInfo, lines below. All info but not info operator wants to be notified about.